src/components/Explorer/fields.js

```diff
@@ -493,6 +500,9 @@ ${props.player && props.player.value ? '' : 'AND player_matches.account_id < pla
       return { text: strings[str], value: laneRoleId, key: String(laneRoleId) };
     }),
     isTi8Team: [{ text: 'Yes', value: true, key: 'true' }],
+    megaWin: [{ text: 'Yes', value: true, key: 'true' }],
+    radiantMinGoldAdvantage: goldAdvantage,
```

Comment:
should probably rename this field to minGoldAdvantage to avoid confusion

Reply:
done